test/acceptance/pageobjects/ffc-demo-claimname.js

```diff
@@ -0,0 +1,29 @@
+import Page from './page'
```

Comment:
If we are not using any of the pageobject files, this whole folder could be removed